randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -2610,106 +2260,25 @@ Extra - asset_id: collision_camera_031
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
-  > Tile Group (SCREWATTACK) 2
-      Trivial
-  > Tile Group (SCREWATTACK) 3
-      Trivial
-
-> Life Recharge; Heals? True; Default Node
-  * Layers: default
-  * Extra - actor_name: energyrecharge_002
-  * Extra - actor_def: actordef:actors/props/energyrecharge/charclasses/energyrecharge.bmsad
-  * Extra - start_point_actor_name: energyrecharge_002_platform
-  * Extra - start_point_actor_def: actordef:actors/props/weightactivatedplatform_energy/charclasses/weightactivatedplatform_energy.bmsad
-  > Top Room
-      Trivial
-
-> Tile Group (BOMB) 1; Heals? False
-  * Layers: default
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_014
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('BOMB',)
-  > Tile Group (POWERBEAM)
-      Morph Ball
-  > Left Water Ledge
-      Morph Ball
-
-> Tile Group (SCREWATTACK) 1; Heals? False
-  * Layers: default
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_015
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('SCREWATTACK',)
-  > Tile Group (BOMB) 1
-      Morph Ball
-  > Tile Group (BOMB) 2
-      Morph Ball
   > Dock to Teleport to Burenia
-      No Gravity Suit or Movement (Beginner)
-  > Left Water Ledge
-      Trivial
-
-> Tile Group (BOMB) 2; Heals? False
-  * Layers: default
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_035
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - actor_layer: breakables
-  * Extra - tile_types: ('BOMB',)
-  > Tile Group (POWERBEAM)
-      Morph Ball
+      All of the following:
+          Screw Attack
+          Any of the following:
+              # Having Gravity Suit makes this harder, so there's a Movement trick to account for this
+              Movement (Beginner) or Enabled Highly Dangerous Logic
```

Comment:
As was discussed in #dev-talk, this can actually just be made trivial with Screw Attack, as gravity alone is enough from below and should be added as such.